src/06-loops.js

```diff
@@ -137,7 +193,14 @@ function firstIndexFound(array, target) {}
  *  lastIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function lastIndexFound(array, target) {}
+function lastIndexFound(array, target) {
+  for (let i = array.length; i > 0; i--) {
```

Comment:
With better testing, this code should not pass. For this loop to accurately do what we intend, i should equal array.length-1 and our while condition should be i >= 0. As its stands now if the last element is first in the array we wont accurately return an index.